bot/tests/test_revisions.py

```diff
@@ -133,3 +133,22 @@ def as_phabricator_lint():
     assert mock_revision.contains(issue_in_existing_file_added_line)
     assert not mock_revision.contains(issue_in_not_changed_file)
     assert mock_revision.contains(issue_full_file)
+
+
+def test_bugzilla_id(mock_revision):
+    """Test the bugzilla id parsing from phabricator data"""
+
+    # Default value from mock
+    assert mock_revision.bugzilla_id == 1234567
+
+    # Update phabricator data on revision
+    mock_revision.revision["fields"]["bugzilla.bug-id"] = "456789"
+    assert mock_revision.bugzilla_id == 456789
+
+    # On bad data fallback gracefully
```

Comment:
Do you have a real example of this?

Reply:
Bad data ? No, but it could happen if a client less smart than moz-phab publishes on phabricator a revision: it's a string there